src/cmap/connection.ts

```diff
@@ -439,7 +439,7 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
         zlibCompressionLevel: this.description.zlibCompressionLevel
       });
 
-      if (options.noResponse) {
+      if (options.noResponse || ('moreToCome' in message && message.moreToCome)) {
```

Comment:
https://github.com/mongodb/node-mongodb-native/blob/NODE-6060/fire-and-forget/src/cmap/commands.ts#L315-L316

Can we do the same trick we applied to OpReply to OpQuery? Just hard code moreToCome to false since it never is something that is used for that protocol, then we don't have to check for property presence.

Reply:
Sure!